docs/markdown/Pkgconfig-module.md

```diff
@@ -52,10 +52,20 @@ keyword arguments.
   `Version:` field. Defaults to the project version if unspecified.
 - `d_module_versions` a list of module version flags used when compiling
    D sources referred to by this pkg-config file
+- `uninstalled_variables` used instead of the `variables` keyword argument, when
+  generating the uninstalled pkg-config file. Since *0.49.0*
 
 Since 0.46 a `StaticLibrary` or `SharedLibrary` object can optionally be passed
 as first positional argument. If one is provided a default value will be
 provided for all required fields of the pc file:
 - `install_dir` is set to `pkgconfig` folder in the same location than the provided library.
 - `description` is set to the project's name followed by the library's name.
 - `name` is set to the library's name.
+
+Since 0.49.0 uninstalled pkg-config files are generated as well. They are
+located in `<build dir>/pkgconfig-uninstalled/`. It is sometimes
+useful to build projects against libraries built by meson without having to
+install them into a prefix. In order to do so, just set
+`PKG_CONFIG_PATH=<builddir>/pkgconfig-uninstalled` before building your
+application. That will cause pkg-config to prefer those `-uninstalled.pc` files
+and find libraries and headers from the meson builddir.
```

Comment:
This should mention that uninstalled is provided on a best-effort basis and that there is no guarantee that it will work for any specific use case (or keep working if it currently does) because there are many ways that uninstalled libs can fail and we can't support all of them. It should instead recommend subprojects or staged install when possible.

Reply:
done.